app/views/candidate_interface/contact_details/review/show.html.erb

```diff
@@ -2,7 +2,7 @@
   :title,
   title_with_error_prefix(t('page_titles.contact_information'), @contact_details_form&.errors&.any?),
 ) %>
-<% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
+<% content_for(:before_content, govuk_back_link_to(application_form_path)) %>
```

Comment:
Minor, minor point but a couple of these are wrapped in `()` and the rest are not?